sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/administration/BuildModel.java

```diff
@@ -13,15 +13,15 @@
 /**
  * Sample to build a model with training data.
  * For instructions on setting up documents for training in an Azure Storage Blob Container, see
- * <a href="https://docs.microsoft.com/azure/cognitive-services/form-recognizer/build-training-data-set#upload-your-training-data">here</a>.
+ * <a href="https://aka.ms/azsdk/formrecognizer/buildtrainingset">here</a>.
  * <p>
  * For this sample, you can use the training documents found in
- * <a href="https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/formrecognizer/azure-ai-formrecognizer/src/samples/resources/sample-forms/training">here</a>
+ * <a href="https://aka.ms/azsdk/formrecognizer/sampletrainingfiles">here</a>
  * to create your own custom document analysis models.
  * For instructions to create a label file for your training forms, please see:
- * <a href="https://docs.microsoft.com/azure/cognitive-services/form-recognizer/label-tool?tabs=v2-1">here</a>.
+ * <a href="https://aka.ms/azsdk/formrecognizer/labelingtool">here</a>.
```

Comment:
Sorry., I am confused. so https://aka.ms/azsdk/formrecognizer/labelingtool targets the label tool v2.1?
and the idea is to create a new link for the new labeling tool? and is the name of that new tool FR studio?

Reply:
Latest API version FR doesn't use the labeling tool/FOTT - it uses FR studio. We could link to both in libraries where we have multiapi support, no strong opinion there. But I do think we should create a new link for FR studio specifically.

Reply:
I see. my confusion was that I left the links that are like `https://docs.microsoft.com/azure/cognitive-services/form-recognizer/label-tool?tabs=v2-1` for v2.1 labeling stuff and started using the `https://aka.ms/azsdk/formrecognizer/labelingtool` for v3.0 so what technically is the FR studio.

What are you doing in Python?

Reply:
I thought we didnt have an aka.ms link for FR studio??

Reply:
I have an issue to address updating our docs which I'm going to be getting to today. If we're missing the aka.ms link I can create it...but let me know if instead I should replace the link the the labelingtool link is pointing to.

Reply:
I don't believe we knew the link to FR Studio at release time, nor did we know that it would replace the labeling tool for V3 (oops). 

AFAIK we don't have a link to FR studio. The labeling tool one points to the labeling tool docs. I think we could leave that one as-is, and create a new https://aka.ms/azsdk/formrecognizer/formrecognizerstudio link.

Reply:
> nor did we know that it would replace the labeling tool for V3 (oops).

Sounds about right.

Also, we need to make sure we change the wording around the link as I kept using Labeling Tool and now I know that I should have used FR Studio :P

Thank you all for the clarifications

Reply:
Thank you everyone, created and updated to use the new https://aka.ms/azsdk/formrecognizer/formrecognizerstudio link